README.md

```diff
@@ -66,24 +66,27 @@ There is a minimal Gradio UI you can use when running the OLS server locally.  T
 
 By default this interface will ask the OLS server to retain and use your conversation history for subsequent interactions.  To disable this behavior, expand the `Additional Inputs` configuration at the bottom of the page and uncheck the `Use history` checkbox.  When not using history each message you submit to OLS will be treated independently with no context of previous interactions.
 
-### In Cluster Deployment
-Deploying OLS on an openshift cluster is fairly easy with the configuration files we have in [manifests](./manifests) folder.
+### Deploying OLS using OpenShift
 
-You can use the existing image built from the latest code via this image pullspec: quay.io/openshift/lightspeed-service-api:latest
+A Helm chart is available for installing the service in OpenShift.
 
-If you need to build your own image, you can use the following commands:
+To install the chart with the release name `ols-release` in the namespace `ols`:
 
+```shell
+helm upgrade --install ols-release helm/ --create-namespace --namespace ols
```

Comment:
Do we want to specify the `ols` namespace as a resource manifest file too? just like we did with `ols-client-test` namespace.

Reply:
I thought about this, but probably we should give the user the opportunity to choose their own namespace.